sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -26,6 +26,16 @@ public class ProcessMessageEventArgs : EventArgs
         /// </summary>
         public CancellationToken CancellationToken { get; }
 
+        /// <summary>
+        /// The entity path corresponding to the entity that the message was received from.
```

Comment:
```suggestion
        /// The path of the Service Bus entity that the message was received from.
```

Based on the [Receiver comment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs#L34)